MapleServer2/PacketHandlers/Game/StatPointHandler.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.Extensions.Logging;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Servers.Game;
+using MapleServer2.Packets;
+using System;
+
+namespace MapleServer2.PacketHandlers.Game
+{
+    public class StatPointHandler : GamePacketHandler
+    {
+        public override RecvOp OpCode => RecvOp.STAT_POINT;
+
+        public StatPointHandler(ILogger<StatPointHandler> logger) : base(logger) { }
+
+        public override void Handle(GameSession session, PacketReader packet)
+        {
+            byte mode = packet.ReadByte();
+
+            switch (mode)
+            {
+                case 2: 
+                // Increment attribute point
+                    HandleStatIncrement(session, packet);
+                    // follow up SendStat packet to refresh updated stats
```

Comment:
comment should go before the previous line or on the previous line after the comment

Reply:
Line 25 is actually an old placeholder for where the SendStat function call would have gone. I'll remove this line since I moved the SendStat function call to the HandleStatIncrement function.

Reply:
Line 25 has been removed.